src/test/java/org/candlepin/test/DatabaseTestFixture.java

```diff
@@ -650,6 +651,20 @@ protected Owner createOwner(String key, String name) {
         return owner;
     }
 
+    protected Owner createOwnerEnt(String key) {
+        return this.createOwnerEnt(key, key);
+    }
+
+    protected Owner createOwnerEnt(String key, String name) {
+        Owner owner = TestUtil.createOwner(key, name);
+        owner.setId(null);
+        owner.setContentAccessMode(ContentAccessMode.ENTITLEMENT.toDatabaseValue());
+
+        this.ownerCurator.create(owner);
```

Comment:
Return the value from create or merge in utility methods like this.

Reply:
done